week-3/Homework/mandatory/2-bush-berries.js

```diff
@@ -10,9 +10,11 @@
   Use the tests to confirm which message to return
 */
 
-function bushChecker() {
-
+function bushChecker(bushes) {
+  let bushBerryColours = bushes.every(Element => Element === "pink");
+  return bushBerryColours >0 ? "Bush is safe to eat from" : "Toxic! Leave bush alone!"
```

Comment:
You're really close but I don't think this is quite right.

What is the type of the value that `.every()` returns? (Hint: try looking at the [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every)). Think carefully about how you would use this value in the ternary operator (like you have here) or an `if` statement.